addon/components/hyper-table/cell-renderers/date.js

```diff
@@ -39,7 +39,7 @@ export default Component.extend({
       this.set("isEditing", false);
     },
     updateDate(date) {
-      this.manager.updateColumnValue(this.column.property, this.item, +date[0]/1000);
+      this.item.set(this.column.property, +date[0]/1000);
```

Comment:
why ??

Reply:
simplifying, updateColumnValue did the same thing `item.set(key, value);`